src/Platform/Commands/LinkCommand.php

```diff
@@ -32,20 +32,23 @@ class LinkCommand extends Command
      */
     public function handle(Dashboard $dashboard)
     {
-        $prefix = public_path('/resources');
+        $prefix = public_path('resources');
 
-        if (file_exists($prefix)) {
+        if (file_exists($prefix) && !$this->option('force')) {
```

Comment:
Hi, I think that `force` should also update the previously created links. 
I suggest simply if you have the option to delete the directory?

```php
if($this->option('force')) {
    $this->getLaravel()->make('files')->deleteDirectory($prefix)
}
```
Then there will be minimum changes and the update will be in any case.

What do you think about it?

Reply:
When deleting via `deleteDirectory`, there is a chance that it will delete the contents of the link (If the source is not from vendor)

Reply:
Do you mean it will not delete the link, but the content that was generated by the user? 
For example, commands like: `artisan preset orchid-source` ?

Reply:
For some reason, if the source of the link is not in the `vendor`, for example, `./packages/platform/public`, then when the `deleteDirectory` is called, the contents of `./packages/platform/public` will also be deleted. (Maybe this is a windows problem)

Reply:
Oh, good. Then let's leave it as it is.
But there is a point that `force` assumes that the link will be replaced, but in reality only new ones are added, while the old ones are not deleted. Can we do something about this?